app/pages/_app.tsx

```diff
@@ -4,34 +4,34 @@ import Head from "next/head";
 import "../styles/globals.scss";
 
 const metaTags = {
-  title: "Auckland DHB Vaccination Rates to 90%",
+  title: "Aotearoa New Zealand Vaccination Rates to 90% - outoflockdown.co",
   description:
-    "Easily view Auckland's vaccinations to 90% by DHB and the daily progress towards getting out of lockdown.",
+    "View vaccination rates for each District Health Board (DHB) in New Zealand and track daily progress towards the 90% goal, enabling Kiwis to get out of lockdown.",
+  url: "https://outoflockdown.co",
+  imageUrl: "https://outoflockdown.co/preview-cover.png",
```

Comment:
Might be worth putting a env var on this - not necessary for now

Reply:
Actually if we just drop the uri:

```suggestion
  imageUrl: "/preview-cover.png",
```

Reply:
Will have to be env var based imo may leave for now. Relative won't work since it's not a resource the user/client resolves when they land on the page but rather something the site that the link being shared on tries to resolve

https://stackoverflow.com/questions/9858577/open-graph-can-resolve-relative-url/9858694#9858694

Reply:
Ah gotcha - That's definitely my bad 🙏 

Let's merge as is, we can add an env var when we're actually battling to debug some stuff